environment_test.sh

```diff
@@ -5,6 +5,7 @@ export NOTIFY_ENVIRONMENT='test'
 export ADMIN_CLIENT_SECRET='dev-notify-secret-key'
 export ADMIN_BASE_URL='http://localhost:6012'
 export FROM_NUMBER='from_number'
+export FREE_SMS_TIER_FRAGMENT_COUNT='250000'
```

Comment:
why is this necessary? FREE_SMS_TIER_FRAGMENT_COUNT is in config.py

Reply:
It wasn't necessary - now removed.